static/js/task/controllers/task.controller.js

```diff
@@ -22,17 +22,22 @@
             self.task_worker_id = $routeParams.taskWorkerId;
             self.task_id = $routeParams.taskId;
 
-            if (!self.task_worker_id) { //if they navigate away mid-queue and then attempt task from task feed
```

Comment:
Executed the same steps but no such issue came up. 
The issue you are getting is because of incorrect id being used to fetch task. See new line 38 which corrects this.